hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterRpc.java

```diff
@@ -2074,9 +2074,6 @@ public void testMkdirsWithCallerContext() throws IOException {
     GenericTestUtils.LogCapturer auditlog =
         GenericTestUtils.LogCapturer.captureLogs(FSNamesystem.AUDIT_LOG);
 
-    // Current callerContext is null
-    assertNull(CallerContext.getCurrent());
```

Comment:
Why can't we check this?

Reply:
I thought it's quite strange to check that CallerContext is null and reset it next line. But in testRealUserPropagationInCallerContext it is not reset, so, agree to revert this.